webextensions/api/extension.json

```diff
@@ -258,7 +258,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/extension/isAllowedFileSchemeAccess",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "12"
```

Comment:
✅ https://github.com/chromium/chromium/commit/c7c401dd5f78685f7ce4a1e1ba0b38f64702c73e in [12.0.706.0](https://github.com/chromium/chromium/releases/tag/12.0.706.0) before 12.0.742.